src/main/java/decodes/cwms/resevapcalc/ResEvap.java

```diff
@@ -233,7 +203,7 @@ public boolean compute(Date currentTime,
             try
             {
             	xout = new BufferedWriter( new FileWriter( xoutfil ) );
-            	_reservoir.setDebugFile(xout);
+            	reservoir.setDebugFile(xout);
```

Comment:
Any particular reason we don't just dump all this in the log?
We can use level checks to decide if something should actually log.